### PR TITLE
Feature/game stats

### PR DESCRIPTION
**Pull request by noahdurbin**
## Type Of Change
- [X] New Feature
- [ ] Debugging
- [X] Refactor/File Clean-up
- [X] Tests
- [ ] Documentation

## Provide A Summary Of Your Change
Added a tie percentage method and it's helper methods to calculate percentage of tie games out of the total number of games. Also refactored some of the things that Kat had mentioned in our meeting this afternoon and changed the create methods into a class method.

## Group Project Check List
- [x] I made test for my code, and made sure they were passing before submitting PR
- [ ] I assigned at least 2 group members to review my PR

### CODE REVIEW

File: lib/game.rb [hunk 1]
```diff
@@ -52,40 +52,37 @@ def self.percentage_away_wins
...
-    end
+    @home_goals > @away_goals
   end
 
   def away_win?
-    if @away_goals > @home_goals
-      true
-    else 
-      false
+    @away_goals > @home_goals
```
Comment (zach-bergman) [inline lib/game.rb:76]: Nice, looks really good just being a one liner

File: lib/game_factory.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class GameFactory
   @@games = []
 
-  def create_games(path)
+  def self.create_games(path)
```
Comment (zach-bergman) [inline lib/game_factory.rb:4]: Thanks for doing this, I'll use this Class method now setting up the StatTracker

File: lib/game_factory.rb [hunk 3]
```diff
@@ -21,4 +21,8 @@ def create_games(path)
   def self.all_games
     @@games
   end
+
+  def self.reset_games
```
Comment (zach-bergman) [inline lib/game_factory.rb:25]: Teardown method! Sweet

**Review by zach-bergman — APPROVED** [review]
Thanks for adding that Class method for ::create_games, I'll pull down from main and start working!

File: lib/game.rb (hunk 1 above)
Comment (austincarrjones) [reply lib/game.rb:76]: well done!

File: lib/game_factory.rb (hunk 3 above)
Comment (austincarrjones) [reply lib/game_factory.rb:25]: I'm stealing this!

**Review by austincarrjones — APPROVED** [review]

**Pull request merged**